.github/workflows/docker-publish.yml

```diff
@@ -46,14 +42,14 @@ jobs:
         uses: docker/build-push-action@v5
         with:
           context: .
-          platforms: ${{ matrix.platform }}
+          platforms: linux/amd64,linux/arm64
```

Comment:
Add a comment here about what you found:
It is important, that we explicitly list the platforms here, and not e.g. as a GitHub actions `matrix`, because otherwise the manifests will be wrong and the images of different platforms will overwrite each other.

Reply:
Good point, done